src/App.jsx

```diff
@@ -1,35 +1,38 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import Articles from "./components/Articles";
+import Header from "./components/Header";
+import Navbar from "./components/Navbar";
+import "bootstrap/dist/css/bootstrap.min.css";
+import PageNavigation from "./components/PageNavigation";
+import { useState } from "react";
+import ItemsPerPage from "./components/ItemsPerPage";
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [pageNumber, setPageNumber] = useState(1);
+  const [totalPage, setTotalPage] = useState(1);
+  const [itemsPerPage, setItemsPerPage] = useState(10);
+  pageNumber > totalPage && setPageNumber(totalPage);
 
   return (
     <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
+      <Header />
+      <Navbar />
+      <ItemsPerPage
```

Comment:
This is getting quite repetitive - see prev note about considering Context as a refactor